imports/components/@primitives/layout/global/index.js

```diff
@@ -157,34 +157,61 @@ export const URL_TITLE_QUERY = gql`
   }
 `;
 
+export const SAVE_DEVICE_REGISTRATION_ID = gql`
+  mutation SaveDeviceRegistrationId($registrationId: String!) {
+    saveDeviceRegistrationId(registrationId: $registrationId) {
+      error
+      code
+      success
+    }
+  }
+`;
+
+export const withSaveDeviceRegistrationId = graphql(SAVE_DEVICE_REGISTRATION_ID, {
+  props: ({ mutate }) => ({
+    saveDeviceRegistrationId: (registrationId) => mutate({
+      variable: { registrationId },
+    }),
+  }),
+});
+
 class GlobalWithoutData extends Component {
 
   static propTypes = {
     dispatch: PropTypes.func.isRequired,
     client: PropTypes.object.isRequired,
+    saveDeviceRegistrationId: PropTypes.func.isRequired,
   }
 
   state = { universalLinkLoading: false }
 
   componentWillMount() {
     if (Meteor.isCordova) {
       document.addEventListener("click", linkListener);
-      document.addEventListener("deviceready", () => {
-        universalLinks.subscribe("universalLinkRoute", this.universalLinkRouting);
-        /* eslint-disable */
-        if(isIOS()) {
-          FCMPlugin.ready();
-        }
-        FCMPlugin.subscribeToTopic('newspring');
-        /* eslint-enable */
-      }, false);
+      document.addEventListener("deviceready", () => this.deviceReadyFunction(this.props.saveDeviceRegistrationId), false);
     }
   }
 
   componentWillUnMount() {
     if (Meteor.isCordova) universalLinks.unsubscribe("universalLinkRoute");
   }
 
+  deviceReadyFunction = (saveDeviceRegistrationId) => {
+    universalLinks.subscribe("universalLinkRoute", this.universalLinkRouting);
+    /* eslint-disable */
```

Comment:
in fact, why do we need to disable? We can add FCMPlugin to the eslint globals section